### ISSUE TITLE
Support for TemaSys plugin

### ISSUE THREAD

**Issue by contra** [HenrikJoreteg/webrtcsupport#4]
https://temasys.atlassian.net/wiki/display/TWPP/WebRTC+Plugins

Seems like it would be simple enough

**Comment by pablogeek** [comment]
Is there any chance to get this working? @contra

**Comment by contra** [comment]
@pablogeek I got it working by not using this library

**Comment by pablogeek** [comment]
@contra Hi! thanks for your response. What example or library are you using?

**Comment by contra** [comment]
@pablogeek No need for a library to do this anymore

**Comment by fippo** [comment]
well, then let's close this.

**Comment by contra** [comment]
@fippo I think you misunderstood - the issue definitely still exists in this project, I just opted to not have it anymore by using different modules.

**Comment by fippo** [comment]
@contra given the list of differences between the free and commercial version of the plugin I do not think that plugging it is the right thing to do.

Besides, you said it's simple enough and did not show code in more than year. Code rules. Note that i'd still ignore PRs for the reason given above.

**Comment by contra** [comment]
- I didn't ask for "plugging it in" i asked for "adding support" aka not blowing up when the browser isn't chrome or firefox. This is a duplicate of #18 essentially. Right now this module completely explodes on a number of platforms because it makes a lot of poor assumptions.
- Where did I say it's "simple enough" and not provide code one year ago? I understand that code rules, I am also an engineer.

I think it's telling that this ticket didn't even receive a response until 5 days ago, even though I filed the issue over a year ago. Like I said in my response, I moved on to other modules *a year ago* after this issue was ignored, so I don't have code to show you. Even though this issue isn't a problem for me anymore, I still think it's wrong for you to close the ticket just because I gave up on complaining. Not fantastic OSS methodology.
